### PR TITLE
Feature/theme update

### PR DESCRIPTION
**Pull request by ThomasZho**
updated architecture

### CODE REVIEW

**Comment by AurelGit** [comment]
Sounds perfect :)

File: App/Theme/index.js [hunk 1]
```diff
@@ -3,5 +3,6 @@ import Fonts from './Fonts'
 import Metrics from './Metrics'
 import Images from './Images'
 import ApplicationStyles from './ApplicationStyles'
+import Helpers from './Helpers'
 
 export { Colors, Fonts, Images, Metrics, ApplicationStyles }
```
Comment (JeremyDolle) [inline App/Theme/index.js]: You need to add Helpers in `export`

File: App/Theme/Colors.js [hunk 2]
```diff
@@ -7,8 +7,12 @@
 
 export default {
   transparent: 'rgba(0,0,0,0)',
-  // Example colors:
+  white: '#FFF',
+  darkWhite: '#FCFCFC',
   text: '#212529',
+  border: '#979797',
```
Comment (JeremyDolle) [inline App/Theme/Colors.js]: You don't need to add unused colors :)

**Comment by AurelGit** [comment]
@ThomasZho could you please move App/Assets/Images/README.md to App/Images/README.md → App/Assets/README.md with an explanation of the directory usage ?

**Pull request merged**